v4/source/emoji.yaml

```diff
@@ -56,3 +56,34 @@
           $ref: '#/responses/Forbidden'
         '501':
           $ref: '#/responses/NotImplemented'
+
+  '/emoji/{emoji_id}/image':
+    get:
+      tags:
+        - emoji
+      summary: Get custom emoji image
+      description: |
+        Get custom emoji image for the team.
+        ##### Permissions
+        Must be authenticated.
+      parameters:
+        - name: emoji_id
+          in: path
+          description: Emoji GUID
+          required: true
+          type: string
+      responses:
+        '200':
+          description: Emoji image retrieve successful
+          schema:
+            type: string
```

Comment:
Can you remove the schema here? I think it's fine to just have the description for this one